glusterd2/xlator/options/options.go

```diff
@@ -187,14 +188,17 @@ func ValidateInt(o *Option, val string) error {
 
 // ValidateInternetAddress validates the Internet Address
 func ValidateInternetAddress(o *Option, val string) error {
+	if len(val) == 0 {
+		return ErrInvalidArg
+	}
 	_, _, err := net.ParseCIDR(val)
```

Comment:
govalidator has a function `IsCIDR`. It will be better to use that to keep validators uniform.